doc/sphinx/internal_calls/brand_portal.rst

```diff
@@ -55,8 +55,8 @@ Once the company has been saved, the domain will be also displayed in the list
 
     .. image:: img/es/domain_list.png
 
-.. attention:: It's important to understand this block. Wihout a DNS domain
-   pointing to our users proxy IP address, everything will fail.
+.. attention:: It's important to understand this block. :ref:`Unless we've a single
+   company registered <dnshack>`, wihout a DNS domain pointing to our users proxy IP address, everything will fail.
```

Comment:
We try to keep documentation lines no longer than 80 columns.

If you can, split this long line into:
```
.. attention:: It's important to understand this block. :ref:`Unless we've a single
￼   company registered <dnshack>`, wihout a DNS domain pointing to our users proxy
   IP address, everything will fail.
```

This may generate a _fuzzy_ translation though.